src/main/java/us/kbase/narrativemethodstore/db/mongo/MongoDynamicRepoDB.java

```diff
@@ -619,6 +635,7 @@ public void saveToStream(OutputStream os)
     // it is actually used, just some of the fields aren't explicitly used. They're populated
     // by Jackson though.
     @SuppressWarnings("unused")
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
https://github.com/kbase/narrative_method_store/actions/runs/11450814158/job/31858967858#step:5:792

Reply:
If my understanding is correct, you deal with this in [cleanObject](https://github.com/kbase/narrative_method_store/blob/9d204a4ff81bb4217f4a7896842f6ed5eaabf566/src/main/java/us/kbase/narrativemethodstore/db/mongo/MongoUtils.java#L128).

However, with this line, along with the MongoDB 7 document, it can handle the conversion, and therefore the cleanObject function can be removed.

Reply:
I see. Is there a way to get it to just ignore the `_id` field rather than doing a blanket ID?

Reply:
Here we want to ignore a property ("_id") that's not in the class, so we can use the ignoreUnknown=true option. We can also specify properties to ignore that are actually present in the class, but "_id" is not in RepoHistory.

Reply:
This seems like it might work https://stackoverflow.com/questions/53389302/use-of-jsonignoreproperties-specific-property-deserialize-properties-exists-only

Reply:
👍

Reply:
:+1: